[2024-04-30T16:56:18.397+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2024-04-30T16:56:09.848256+00:00 [queued]>
[2024-04-30T16:56:18.403+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2024-04-30T16:56:09.848256+00:00 [queued]>
[2024-04-30T16:56:18.403+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-04-30T16:56:18.404+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-04-30T16:56:18.404+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-04-30T16:56:18.411+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2024-04-30 16:56:09.848256+00:00
[2024-04-30T16:56:18.415+0000] {standard_task_runner.py:55} INFO - Started process 765 to run task
[2024-04-30T16:56:18.419+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2024-04-30T16:56:09.848256+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpsu2r5d50']
[2024-04-30T16:56:18.421+0000] {standard_task_runner.py:83} INFO - Job 19: Subtask store_user
[2024-04-30T16:56:18.467+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user manual__2024-04-30T16:56:09.848256+00:00 [running]> on host ec0cac6d8211
[2024-04-30T16:56:18.513+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2024-04-30T16:56:09.848256+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-04-30T16:56:09.848256+00:00
[2024-04-30T16:56:18.514+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2024-04-30T16:56:18.522+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-04-30T16:56:18.527+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-04-30T16:56:18.536+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20240430T165609, start_date=20240430T165618, end_date=20240430T165618
[2024-04-30T16:56:18.550+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-04-30T16:56:18.563+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
